internal/quickstart/choose_sdk.go

```diff
@@ -61,15 +62,84 @@ func NewChooseSDKModel(selectedIndex int) tea.Model {
 			Quit:          BindingQuit,
 		},
 		list:          l,
+		sdkDetails:    sdkDetails,
 		selectedIndex: selectedIndex,
 	}
 }
 
-// initSDKs sets the index of each SDK based on place in list.
-func initSDKs() {
-	for i := range SDKs {
-		SDKs[i].index = i
+// The CLI uses the sdkmeta project to obtain metadata about each SDK, including the display names
+// and types (client, server, etc.)
+// Currently, there is no sdkmeta for code examples associated with each SDK, so we hard-code the examples here.
+// Once they are part of sdkmeta we can remove this list.
+var sdkExamples = map[string]string{
+	"react-client-sdk":  "https://github.com/launchdarkly/react-client-sdk/tree/main/examples/typescript",
+	"vue":               "https://github.com/launchdarkly/vue-client-sdk/tree/main/example",
+	"react-native":      "https://github.com/launchdarkly/js-core/tree/main/packages/sdk/react-native/example",
+	"cpp-client-sdk":    "https://github.com/launchdarkly/cpp-sdks/tree/main/examples/hello-cpp-client",
+	"cpp-server-sdk":    "https://github.com/launchdarkly/cpp-sdks/tree/main/examples/hello-cpp-server",
+	"lua-server-server": "https://github.com/launchdarkly/lua-server-sdk/tree/main/examples/hello-lua-server",
+}
+
+// sdkOrder is a list of IDs the SDKs in order that they should be rendered based on popularity.
+var sdkOrder = []string{
```

Comment:
I added back the order in which the SDKs should show up based on popularity. Otherwise they show up in alphabetical order.

Reply:
Ah, that's interesting - I guess sdk-meta needs to maintain a popularity list as well. I'll make a ticket.